backend/api/Services/RobotService.cs

```diff
@@ -268,7 +268,7 @@ public async Task<IList<Robot>> ReadRobotsForInstallation(string installationCod
                 .ToListAsync();
         }
 
-        public async Task SetRobotOffline(string robotId)
+        public async Task SetRobotToIsarDisconnected(string robotId)
```

Comment:
This function does more than it says, we split the functionality or set the function to explicitly state what it does

Reply:
Handled by issue https://github.com/equinor/flotilla/issues/1493